### PR TITLE
Fix/sentence quality

### PR DESCRIPTION
**Pull request by ozgurnsahin**
Created _process_regex function to further improve sentence quality of the extracted regex

### CODE REVIEW

File: functions/reading_functions.py [hunk 1]
```diff
@@ -59,9 +63,39 @@ def read_file(self, file_path: str):
...
         docs = self.nlp(text)
         sentences = [sent.text.replace('\n', ' ').strip() for sent in docs.sents]
-        valid_sentences = [sentence for sentence in sentences if len(sentence) > 15]
+        valid_sentences = [sentence for sentence in sentences if len(sentence) > 10]
         file_data["page_sentence_amount"].append(len(valid_sentences))
-        file_data["sentences"].extend(valid_sentences)
\ No newline at end of file
+        file_data["sentences"].extend(valid_sentences)
+
+    # #query spell handler
```
Comment (rahmansahinler1) [inline functions/reading_functions.py:85]: Delete commented out code

**Comment by rahmansahinler1** [comment]
1- change resource branch from main to develop
2- add yourself as assignee
3- attach label
4- attach project

File: .gitignore [hunk 2]
```diff
@@ -6,7 +6,14 @@ __pycache__/
 
 # C extensions
 *.so
-
+# pdf files
```
Comment (rahmansahinler1) [inline .gitignore:9]: No need explanation here. Just file is enough

File: .gitignore [hunk 3]
```diff
@@ -6,7 +6,14 @@ __pycache__/
...
+*.pdf
+#docx files
+*.docx
+#jupyter notabooks
```
Comment (rahmansahinler1) [inline .gitignore:13]: JUPYTER NOTABOOKS!!

**Review by rahmansahinler1 — CHANGES_REQUESTED** [review]

**Comment by rahmansahinler1** [comment]
eski oldugu icin kapatiyorum guncel mr bu kisimlari kapsiyor sanirim

**Comment by ozgurnsahin** [comment]
Kapsıyor bunu eklemiştik diğer pull requestlere

**Pull request closed without merging**